src/main/resources/io/sealights/plugins/sealightsjenkins/buildsteps/cli/SealightsCLIBuildStep/config.jelly

```diff
@@ -16,14 +16,12 @@
                                  selected="${currentDescriptor == instance.commandMode.descriptor || (instance.commandMode.descriptor == null &amp;&amp; currentDescriptor.default)}"
                                  staplerClass="${currentDescriptor.clazz.name}">
                 <j:set var="instance" value="${instance.commandMode}"/>
-                <j:set var="descriptor" value="${instance.descriptor}"/>
```

Comment:
maybe its worth adding documentation on the research and the results we discovered